guiprep.pl

```diff
@@ -1297,38 +1220,37 @@
     	foreach $drv(@drivelist) {$drives->insert("end", $drv)};
 }
 
-my $p4lframe =  $page4->Frame()->pack(-side => 'top', -anchor => 'n');
+# TODO: END HERE
```

Comment:
???

Reply:
Yep, that's just an omission -- to get this to work piecemeal I had most of the stuff commented out and just worked my way down until it loaded successfully. Need to remove.

Reply:
Fixed with ad76a9b.